webapp/niapp.yaml

```diff
@@ -416,3 +442,68 @@ definitions:
     format: binary
     title: WebApp Content
     description: The webapp binary content. Depending on the webapp's type it can be a dashboard, template or *.nipkg file exported from LabVIEW NXG
+  WebAppsAdvancedQuery:
+    type: object
+    title: Advanced Query Object for WebApps
+    properties:
+      filter:
+        description: >-
+            The filter criteria for webapps, consisting of a string of queries composed using AND/OR operators.
+            String values need to be enclosed in double quotes.
+            Parenthesis can be used within the filter to better define the order of operations.
+
+
+            Filter syntax: '[property name][operator][operand] and [property name][operator][operand]'
+
+            Operators:
+
+            - Equals operator '='. Example: 'x = y'
+
+            - Not equal operator '!='. Example: 'x != y'
+
+            - Greater than operator '>'. Example: 'x > y'
+
+            - Greater than or equal operator '>='. Example: 'x >= y'
+
+            - Less than operator '<'. Example: 'x < y'
+
+            - Less than or equal operator '<='. Example: 'x <= y'
+
+            - Logical AND operator 'and'. Example: 'x and y'
+
+            - Logical OR operator 'or'. Example: 'x or y'
+
+            - Contains operator '.Contains()', used to check whether a string contains another string. Example: 'x.Contains(y)'
+
+            - Does not contain operator '!.Contains()', used to check whether a string does not contain another string. Example: '!x.Contains(y)'
+
+            - String null or empty 'string.IsNullOrEmpty()', used to check whether a string is null or empty. Example: 'string.IsNullOrEmpty(x)'
+
+            - String is not null or empty '!string.IsNullOrEmpty()', used to check whether a string is not null or empty. Example: '!string.IsNullOrEmpty(x)'
+
+
+            Valid webapp properties that can be used in the filter:
+
+            - id
+
+            - name
+
+            - properties
+
+            - shared
+
+            - type
+
+            - workspace
+        type: string
+        example: name.Contains("myWebApp") || type == "WebVI"
+      take:
+        description: The maximum number of webapps to return
+        type: integer
+        default: 100
+        minimum: 0
+        maximum: 1000
+        example: 10
+      continuationToken:
```

Comment:
This should contain an "example" element, otherwise the continuation token will be displayed in the swagger editor as "string".

Reply:
I wasn't sure what example to use, so I used "token" to be consistent with the test monitor swagger doc.